unity/Assets/0Refactored/scenes/laboratory/Scripts/scrSegmentBuilder.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using GEAR.Serialize;
+
+public class scrSegmentBuilder : MonoBehaviour
+{
+    // #################################################################################################################
+    // Members
+
+    // +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    // Given Segments
+    public GameObject staticSegementStart;
+    public GameObject staticSegmentEnd;
+    public GameObject staticSegmentContainer;
+
+    // +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    // Prefabs
+    public GameObject prefabSegment;    
+    public GameObject prefabEmptyPreview;
+
+    // +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    // Previews
+    public GameObject[] prefabsExperimentPreview;
+
+    // +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    // Counters
+    private int numberSegmentsTotal;
+    private int numberSegmentsCurrent = 0;
+
+    // #################################################################################################################
+    // Methods
+
+    // +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    // Start is called before the first frame update
+    void Start()
+    {
+        this.buildAllSegments();
+    }
+
+    // +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    // Builds all segments according to the members
+    void buildAllSegments()
```

Comment:
I think we should introduce naming conventions. Usually, methods start with a capital letter.